### DISCUSSION TITLE
8bitdo SNES30 issue where control pad isn't working.

### DISCUSSION THREAD

**Discussion by ghost** [darthcloud/BlueRetro#997] [General]
I tried synching this controller (SNES30 - the one without joysticks) via x-input mode onto the BlueRetro receiever on my SNES (real hardware) and all the buttons on the controller seem to be working except for the control pad.  This controller is using the latest firmware v4.20 which is also the same firmware used on the SN30 & SF30 retro game pad controllers.  I read on here that trying to press select and press up should fix the issue but it doesn't.  Tried doing this twice, with the controller on and off.  I even updated the BlueRetro receiver to the latest (SNES.bin) firmware that was released [24.04] 2024-04-24 and it's still has this issue.  I know it's not the controller problem because the controller is working fine on my other android devices.

Lastly, I even tried logging in the reciever (via BT) to play around with the settings and nothing seems to make the control pad work.  Any help to make this controller work would be appreciated.  Thanks.

**Comment by DoctorHeadlock** [comment]
Rebind the very top four directional inputs to their corresponding d-pad inputs on the SNES label. For some reason, the original 8bitdo SN30 maps its d-pad as an analogue stick in x-input mode.
